settings.yaml

```diff
@@ -5,30 +5,30 @@
 ############################
 
 # scenario defs
-region: austin
+region: sfbay
 scenario: base
 start_year: 2010
-end_year: 2012
+end_year: 2013
 land_use_freq: 1
 travel_model_freq: 1
 
 # simulation platforms (leave blank to turn off)
 land_use_model: urbansim
-travel_model: polaris
-activity_demand_model:
+travel_model: beam
+activity_demand_model: activitysim
 
 # docker settings
 docker_images:
-  urbansim: mxndrwgrdnr/block_model_v2_pb
+  urbansim: mxndrwgrdnr/block_model_update
   beam: dimaopen/beam:0.8.6.3
-  activitysim: mxndrwgrdnr/activitysim
+  activitysim: mxndrwgrdnr/activitysim:latest
   polaris:
 docker_stdout: False
 pull_latest: False
 
-# misc pilates settings
-skims_fname: campo_skims.hdf5
-# skims_fname: 10.activitySimODSkims.UrbanSim.TAZ.Full.csv.gz
+# skim settings
+skims_zone_type: taz  # one of [taz, block_group]
```

Comment:
zone type can be different for different regions

Reply:
Right. That's why I have it in the settings.yaml, which the place where users are supposed to make region-specific changes.